wp-content/themes/csisjti/assets/_scss/blocks/_jti-team.scss

```diff
@@ -0,0 +1,124 @@
+@use '../abstracts' as *;
+
+.csis-block--teams {
+  margin-top: rem(32px);
+  text-align: center;
+
+  @include breakpoint('medium') {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    column-gap: rem(32px);
+  }
+
+  @include breakpoint('large') {
+    column-gap: rem(48px);
+  }
+
+  .teams {
+    &__team {
+      margin-bottom: rem(48px);
+
+      &:last-child {
+        margin-bottom: rem(40px);
+      }
+
+      @include breakpoint('large') {
+        text-align: left;
+      }
+
+      h3 {
+        @extend %font-ui-text-lg;
+        margin-bottom: rem(24px);
+        padding-bottom: rem(8px);
+        color: $color-black-100;
+        text-transform: uppercase;
+        border-bottom: 1px solid $color-gray-line;
+
+        @include breakpoint('medium') {
+          display: table-cell;
+          height: rem(72px); // Added this to get the headings to line up
+          text-align: left;
+          vertical-align: bottom;
+        }
+      }
+      
+      span {
+        color: $color-black-155;
+        font-family: $font-barlow;
+        font-weight: 400;
+        @include font-size(17px);
+        font-style: italic;
+        line-height: 1.15;
+        text-transform: initial;
+      }
+    }
+
+    &__list {
+      margin-left: 0;
+
+      @include breakpoint('large') {
+        margin-bottom: 0;
+      }
+    }
+
+    &__item {
+      &:not(:last-child) {
+        margin-bottom: rem(32px);
+      }
+
+      @include breakpoint('large') {
+        display: grid;
+        grid-template-columns: 1fr 2.5fr;
```

Comment:
In general, I don't think you can use decimal places in `fr` units (I think). Let's sub this out for `100px auto`.

We also need to add `grid-template-rows: max-content max-content auto;` to make the spacing consistent between the grid rows.